Traceback (most recent call last):
  File "C:\ProgramData\Miniconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\ProgramData\Miniconda3\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\ProgramData\Miniconda3\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\ProgramData\Miniconda3\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\ProgramData\Miniconda3\lib\asyncio\base_events.py", line 647, in run_until_complete
    return future.result()
  File "C:\ProgramData\Miniconda3\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\ProgramData\Miniconda3\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\ProgramData\Miniconda3\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
%%stata
  import delimited http://www-eio.upc.edu/~pau/cms/rdata/csv/Ecdat/Cigar.csv, clear
  xtset state year

------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mcom_error[0m                                 Traceback (most recent call last)
File [1;32mC:\ProgramData\Miniconda3\lib\site-packages\win32com\client\dynamic.py:86[0m, in [0;36m_GetGoodDispatch[1;34m(IDispatch, clsctx)[0m
[0;32m     85[0m [38;5;28;01mtry[39;00m:
[1;32m---> 86[0m     IDispatch [38;5;241m=[39m [43mpythoncom[49m[38;5;241;43m.[39;49m[43mconnect[49m[43m([49m[43mIDispatch[49m[43m)[49m
[0;32m     87[0m [38;5;28;01mexcept[39;00m pythoncom[38;5;241m.[39mole_error:

[1;31mcom_error[0m: (-2147221021, 'Operation unavailable', None, None)

During handling of the above exception, another exception occurred:

[1;31mcom_error[0m                                 Traceback (most recent call last)
Cell [1;32mIn[2], line 1[0m
[1;32m----> 1[0m [43mget_ipython[49m[43m([49m[43m)[49m[38;5;241;43m.[39;49m[43mrun_cell_magic[49m[43m([49m[38;5;124;43m'[39;49m[38;5;124;43mstata[39;49m[38;5;124;43m'[39;49m[43m,[49m[43m [49m[38;5;124;43m'[39;49m[38;5;124;43m'[39;49m[43m,[49m[43m [49m[38;5;124;43m'[39;49m[38;5;124;43m  import delimited http://www-eio.upc.edu/~pau/cms/rdata/csv/Ecdat/Cigar.csv, clear[39;49m[38;5;130;43;01m\n[39;49;00m[38;5;124;43m  xtset state year[39;49m[38;5;130;43;01m\n[39;49;00m[38;5;124;43m'[39;49m[43m)[49m

File [1;32mC:\ProgramData\Miniconda3\lib\site-packages\IPython\core\interactiveshell.py:2493[0m, in [0;36mInteractiveShell.run_cell_magic[1;34m(self, magic_name, line, cell)[0m
[0;32m   2491[0m [38;5;28;01mwith[39;00m [38;5;28mself[39m[38;5;241m.[39mbuiltin_trap:
[0;32m   2492[0m     args [38;5;241m=[39m (magic_arg_s, cell)
[1;32m-> 2493[0m     result [38;5;241m=[39m fn([38;5;241m*[39margs, [38;5;241m*[39m[38;5;241m*[39mkwargs)
[0;32m   2495[0m [38;5;66;03m# The code below prevents the output from being displayed[39;00m
[0;32m   2496[0m [38;5;66;03m# when using magics with decorator @output_can_be_silenced[39;00m
[0;32m   2497[0m [38;5;66;03m# when the last Python token in the expression is a ';'.[39;00m
[0;32m   2498[0m [38;5;28;01mif[39;00m [38;5;28mgetattr[39m(fn, magic[38;5;241m.[39mMAGIC_OUTPUT_CAN_BE_SILENCED, [38;5;28;01mFalse[39;00m):

File [1;32mC:\ProgramData\Miniconda3\lib\site-packages\ipystata\ipystata_magic.py:320[0m, in [0;36miPyStataMagic.stata[1;34m(self, line, cell, local_ns)[0m
[0;32m    318[0m time[38;5;241m.[39msleep([38;5;241m0.5[39m)
[0;32m    319[0m [38;5;28mself[39m[38;5;241m.[39mlog_dict[session_id] [38;5;241m=[39m os[38;5;241m.[39mpath[38;5;241m.[39mjoin([38;5;28mself[39m[38;5;241m.[39m_lib_dir, [38;5;124m'[39m[38;5;124mlog_[39m[38;5;132;01m%s[39;00m[38;5;124m.txt[39m[38;5;124m'[39m [38;5;241m%[39m session_id)
[1;32m--> 320[0m [38;5;28mself[39m[38;5;241m.[39msession_dict[session_id] [38;5;241m=[39m [43mwin32com[49m[38;5;241;43m.[39;49m[43mclient[49m[38;5;241;43m.[39;49m[43mDispatch[49m[43m([49m[38;5;124;43m"[39;49m[38;5;124;43mstata.StataOLEApp[39;49m[38;5;124;43m"[39;49m[43m)[49m
[0;32m    321[0m [38;5;28mself[39m[38;5;241m.[39mdo_dict[session_id] [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39msession_dict[session_id][38;5;241m.[39mDoCommandAsync
[0;32m    322[0m [38;5;28mself[39m[38;5;241m.[39msession_dict[session_id][38;5;241m.[39mUtilShowStata([38;5;241m1[39m)

File [1;32mC:\ProgramData\Miniconda3\lib\site-packages\win32com\client\__init__.py:117[0m, in [0;36mDispatch[1;34m(dispatch, userName, resultCLSID, typeinfo, UnicodeToString, clsctx)[0m
[0;32m    115[0m [38;5;250m[39m[38;5;124;03m"""Creates a Dispatch based COM object."""[39;00m
[0;32m    116[0m [38;5;28;01massert[39;00m UnicodeToString [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m, [38;5;124m"[39m[38;5;124mthis is deprecated and will go away[39m[38;5;124m"[39m
[1;32m--> 117[0m dispatch, userName [38;5;241m=[39m [43mdynamic[49m[38;5;241;43m.[39;49m[43m_GetGoodDispatchAndUserName[49m[43m([49m[43mdispatch[49m[43m,[49m[43m [49m[43muserName[49m[43m,[49m[43m [49m[43mclsctx[49m[43m)[49m
[0;32m    118[0m [38;5;28;01mreturn[39;00m __WrapDispatch(dispatch, userName, resultCLSID, typeinfo, clsctx[38;5;241m=[39mclsctx)

File [1;32mC:\ProgramData\Miniconda3\lib\site-packages\win32com\client\dynamic.py:106[0m, in [0;36m_GetGoodDispatchAndUserName[1;34m(IDispatch, userName, clsctx)[0m
[0;32m    103[0m     [38;5;66;03m## ??? else userName remains None ???[39;00m
[0;32m    104[0m [38;5;28;01melse[39;00m:
[0;32m    105[0m     userName [38;5;241m=[39m [38;5;28mstr[39m(userName)
[1;32m--> 106[0m [38;5;28;01mreturn[39;00m ([43m_GetGoodDispatch[49m[43m([49m[43mIDispatch[49m[43m,[49m[43m [49m[43mclsctx[49m[43m)[49m, userName)

File [1;32mC:\ProgramData\Miniconda3\lib\site-packages\win32com\client\dynamic.py:88[0m, in [0;36m_GetGoodDispatch[1;34m(IDispatch, clsctx)[0m
[0;32m     86[0m         IDispatch [38;5;241m=[39m pythoncom[38;5;241m.[39mconnect(IDispatch)
[0;32m     87[0m     [38;5;28;01mexcept[39;00m pythoncom[38;5;241m.[39mole_error:
[1;32m---> 88[0m         IDispatch [38;5;241m=[39m [43mpythoncom[49m[38;5;241;43m.[39;49m[43mCoCreateInstance[49m[43m([49m
[0;32m     89[0m [43m            [49m[43mIDispatch[49m[43m,[49m[43m [49m[38;5;28;43;01mNone[39;49;00m[43m,[49m[43m [49m[43mclsctx[49m[43m,[49m[43m [49m[43mpythoncom[49m[38;5;241;43m.[39;49m[43mIID_IDispatch[49m
[0;32m     90[0m [43m        [49m[43m)[49m
[0;32m     91[0m [38;5;28;01melse[39;00m:
[0;32m     92[0m     [38;5;66;03m# may already be a wrapped class.[39;00m
[0;32m     93[0m     IDispatch [38;5;241m=[39m [38;5;28mgetattr[39m(IDispatch, [38;5;124m"[39m[38;5;124m_oleobj_[39m[38;5;124m"[39m, IDispatch)

[1;31mcom_error[0m: (-2147221164, 'Class not registered', None, None)
com_error: (-2147221164, 'Class not registered', None, None)

